_init/_init.1.js

```diff
@@ -0,0 +1,78 @@
+const User = require("../models/user");
+const Document = require("../models/document");
+
+const USERNO = 5;
+const DOCNO = 20;
+
+const userTemplate = JSON.stringify({
+  firstName: "Firstname${index}",
+  lastName: "Lastname${index}",
+  email: "email${index}@gmail.com",
+  password: "abcdef${index}"
+});
+
+const docTemplate = JSON.stringify({
+  owner: "${userId}",
+  authors: ["${userId}"],
+  title: "title${index}",
+  content: "content ${index}"
+});
+
+const makeObjectString = (strTempl, variables) => {
+  let result = strTempl;
+  for (let key in variables) {
+    result = result.replace(new RegExp(`\\$\\{${key}\\}`, "g"), variables[key]);
+  }
+  return result;
+};
+
+const makeArrayOfPromises = (model, data) =>
+  data.map(item =>
+    model
+      .create(item)
+      .then(created => {
+        return created._id;
+      })
+      .catch(err => {
+        return null;
+      })
+  );
+
+async function initDb(db) {
+  let makeUserPromiseList = Promise.resolve(db.dropDatabase())
```

Comment:
You can use `away` here and get results into variable instead of creating intermediate variables.